tests/app/helpers.py

```diff
@@ -550,15 +550,20 @@ def assert_single_question_page_validation_errors(self,
         masthead_heading = doc.xpath('normalize-space(//h2[@class="govuk-error-summary__title"]/text())')
         masthead_link_text = doc.xpath(
             "normalize-space(//ul[contains(@class, 'govuk-error-summary__list')]/li/a/text())")
-        validation_text = doc.xpath('normalize-space(//span[@class="validation-message"]/text())')
+        error_message_span = doc.xpath('//span[@class="govuk-error-message"]')
+        # TODO: When all error messages are govuk-frontend, we can simplify this
```

Comment:
Is there a story tracking this tech debt?

Reply:
There is now: https://trello.com/c/uaAcGw7T/1555-functional-tests-remove-dual-design-system-frontend-toolkit-tests-once-all-apps-have-been-migrated-to-design-system